oxidation/libparsec/crates/crypto/src/rustcrypto/secret.rs

```diff
@@ -13,6 +15,32 @@ use crate::CryptoError;
 
 type Blake2bMac40 = Blake2bMac<U5>;
 
+/// Memory block size in bytes
+const BLOCKSIZE: u32 = 1024;
+/// The maximum amount of RAM that the functions in this module will use, in bytes.
+// https://github.com/sodiumoxide/sodiumoxide/blob/master/libsodium-sys/src/sodium_bindings.rs#L128
+const MEMLIMIT_INTERACTIVE: u32 = 33554432;
```

Comment:
```suggestion
const MEMLIMIT_INTERACTIVE: u32 = 33_554_432;
```
Use `_` to make the number more readable